Adding metadata tag to indicate compliance (#1915)
MicrosoftDocs/VBA-Docs: outlook/Concepts/Getting-Started/customizing-outlook-using-com-add-ins.md

ms.assetid: 84a4f616-3ace-0139-57d5-f0c070064ab2
ms.date: 06/08/2019
ms.localizationpriority: medium

> ms.custom: required-for-compliance # Indicates that this page is linked to in a legal document. 

Reviewer: ```suggestion
ms.custom: required-for-compliance # Indicates that this topic is linked to from a legal document. 
```
